stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -31,7 +31,15 @@ mul!(C::AbstractArray, X::AbstractArray, s::Number) = generic_mul!(C, s, X)
 """
     rmul!(A::AbstractArray, b::Number)
 
-Scale an array `A` by a scalar `b` overwriting `A` in-place.
+Scale an array `A` by a scalar `b` overwriting `A` in-place.  Use
+[`lmul!`](@ref) to multiply scalar from left.  The scaling operation
+respects the semantics of the multiplication [`*`](@ref) between an
+element of `A` and `b`.  In particular, `NaN` entries in `A` are not
+modified (unless the type of `b` defines a non-standard behavior).
+
+!!! compat "Julia 1.1"
+    Prior to Julia 1.1, how `NaN` entries in `A` were treated was
```

Comment:
Maybe rephrase to something like `Prior to Julia 1.1, `NaN` entries in `A` were treated inconsistently.`

Reply:
Thanks, that's much smoother.  I changed the docstring.  I also mention `Inf` since that's also changed.